services/validateAPI/main.py

```diff
@@ -0,0 +1,226 @@
+from fastapi import FastAPI, UploadFile, File, Form
+from fastapi.responses import JSONResponse
+from fastapi.middleware.cors import CORSMiddleware
+from pathlib import Path
+from utils import (
+    find_invalid_country_rows,
+    save_upload_to_tempfile,
+    run_p0_validation,
+    parse_errors,
+    fix_sample_id_spaces,
+    find_taxa_zero_rows,
+    find_invalid_pmi
+)
+import tempfile
+import os
+import logging
+
+logger = logging.getLogger("taxon.validate")
+logging.basicConfig(
+    level=logging.INFO,  # change to DEBUG when debugging
+    format="%(asctime)s | %(levelname)s | %(name)s | %(message)s",
+)
+
+
+app = FastAPI(title="TAXON p0 validator API")
```

Comment:
I'm not sure where this is used but please set the title to "Taxodactyl input validation API"